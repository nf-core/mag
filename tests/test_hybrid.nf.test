nextflow_pipeline {

    name "Test pipeline with test_hybrid profile"
    script "../main.nf"
    tag "pipeline"
    tag "test_hybrid"
    profile "test_hybrid"

    test("-profile hybrid") {

        when {
            params {
                outdir = "$outputDir"
            }
        }

        then {

            // Output directory: all
            def stable_names = getAllFilesFromDir(
                params.outdir, include: ['multiqc/**'], relative: true, includeDir: false
            )

            // Stable contents: Annotation
            def stable_contents_annotation = getAllFilesFromDir(
                params.outdir, include: ['Annotation/**'], ignoreFile: 'tests/.nftignore'
            )

            // Stable contents: Assembly
            def stable_contents_assembly = getAllFilesFromDir(
                params.outdir, include: ['Assembly/**'], ignoreFile: 'tests/.nftignore'
            )

            // Stable contents: GenomeBinning
            def stable_contents_genomebinning = getAllFilesFromDir(
                params.outdir, include: ['GenomeBinning/**'], ignoreFile: 'tests/.nftignore'
            )

            // Stable contents: MultiQC
            def stable_contents_multiqc = getAllFilesFromDir(
                params.outdir, include: ['multiqc/**'], ignoreFile: 'tests/.nftignore'
            )

            // Stable contents: QC Long Reads
            def stable_contents_qclongreads = getAllFilesFromDir(
                params.outdir, include: ['QC_longreads/**'], ignoreFile: 'tests/.nftignore'
            )

            // Stable contents: QC Short Reads
            def stable_contents_qcshortreads = getAllFilesFromDir(
                params.outdir, include: ['QC_shortreads/**'], ignoreFile: 'tests/.nftignore'
            )

            assert workflow.success
            assertAll(
                { assert snapshot(
                    // Number of successful tasks
                    workflow.trace.succeeded().size(),
                    // pipeline versions.yml file for multiqc from which Nextflow version is removed because we test pipelines on multiple Nextflow versions
                    removeNextflowVersion("$outputDir/pipeline_info/nf_core_mag_software_mqc_versions.yml"),
                    stable_names,
                ).match('all') },
                { assert snapshot(stable_contents_annotation).match('Annotation') },
                { assert snapshot(stable_contents_assembly).match('Assembly') },
                { assert snapshot(stable_contents_genomebinning).match('GenomeBinning') },
                { assert snapshot(stable_contents_multiqc).match('multiqc') },
                { assert snapshot(stable_contents_qclongreads).match('QC_longreads') },
                { assert snapshot(stable_contents_qcshortreads).match('QC_shortreads') },
            )
        }
    }
}
