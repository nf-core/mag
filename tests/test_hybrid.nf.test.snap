{
    "-profile hybrid": {
        "content": [
            51,
            {
                "ADJUST_MAXBIN2_EXT": {
                    "coreutils": null
                },
                "BIN_SUMMARY": {
                    "python": "3.10.6",
                    "pandas": "1.4.3"
                },
                "BOWTIE2_ASSEMBLY_ALIGN": {
                    "bowtie2": "2.4.2",
                    "samtools": 1.11,
                    "pigz": "2.3.4"
                },
                "BOWTIE2_ASSEMBLY_BUILD": {
                    "bowtie2": "2.4.2"
                },
                "BOWTIE2_HOST_REMOVAL_ALIGN": {
                    "bowtie2": "2.4.2"
                },
                "BOWTIE2_HOST_REMOVAL_BUILD": {
                    "bowtie2": "2.4.2"
                },
                "BOWTIE2_PHIX_REMOVAL_ALIGN": {
                    "bowtie2": "2.4.2"
                },
                "BOWTIE2_PHIX_REMOVAL_BUILD": {
                    "bowtie2": "2.4.2"
                },
                "BUSCO_BUSCO": {
                    "busco": "6.0.0"
                },
                "BUSCO_UNTAR": {
                    "untar": 1.34
                },
                "CHOPPER": {
                    "chopper": "0.8.0"
                },
                "CONCAT_BINQC_TSV": {
                    "csvtk": "0.31.0"
                },
                "CONVERT_DEPTHS": {
                    "bioawk": 20110810
                },
                "FASTP": {
                    "fastp": "0.24.0"
                },
                "FASTQC_RAW": {
                    "fastqc": "0.12.1"
                },
                "FASTQC_TRIMMED": {
                    "fastqc": "0.12.1"
                },
                "GUNZIP_BINS": {
                    "gunzip": 1.13
                },
                "GUNZIP_SHORTREAD_ASSEMBLIES": {
                    "gunzip": 1.13
                },
                "MAG_DEPTHS": {
                    "python": "3.6.7",
                    "pandas": "1.1.5"
                },
                "MAG_DEPTHS_SUMMARY": {
                    "python": "3.10.6",
                    "pandas": "1.4.3"
                },
                "MAXBIN2": {
                    "maxbin2": "2.2.7"
                },
                "METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS_SHORTREAD": {
                    "metabat2": 2.15
                },
                "METABAT2_METABAT2": {
                    "metabat2": 2.17
                },
                "METASPADESHYBRID": {
                    "spades": "4.1.0"
                },
                "MINIMAP2_HOST_ALIGN": {
                    "minimap2": "2.29-r1283",
                    "samtools": 1.21
                },
                "MINIMAP2_HOST_INDEX": {
                    "minimap2": "2.29-r1283"
                },
                "NANOPLOT_FILTERED": {
                    "nanoplot": "1.44.1"
                },
                "NANOPLOT_RAW": {
                    "nanoplot": "1.44.1"
                },
                "POOL_LONG_READS": {
                    "cat": 9.5
                },
                "PORECHOP_PORECHOP": {
                    "porechop": "0.2.4"
                },
                "PRODIGAL": {
                    "prodigal": "2.6.3",
                    "pigz": 2.6
                },
                "PROKKA": {
                    "prokka": "1.14.6"
                },
                "QUAST": {
                    "python": "3.7.6",
                    "metaquast": "5.0.2"
                },
                "QUAST_BINS": {
                    "python": "3.7.6",
                    "metaquast": "5.0.2"
                },
                "QUAST_BINS_SUMMARY": {
                    "sed": 4.7
                },
                "SAMTOOLS_HOSTREMOVED_INDEX": {
                    "samtools": 1.21
                },
                "SAMTOOLS_HOSTREMOVED_STATS": {
                    "samtools": 1.21
                },
                "SAMTOOLS_HOSTREMOVED_UNMAPPED": {
                    "samtools": 1.21
                },
                "SEQKIT_STATS": {
                    "seqkit": "2.9.0"
                },
                "SPLIT_FASTA": {
                    "python": "3.6.7",
                    "biopython": "1.7.4",
                    "pandas": "1.1.5"
                },
                "Workflow": {
                    "nf-core/mag": "v4.1.0dev"
                }
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.6"
        },
        "timestamp": "2025-09-05T10:57:49.768381226"
    },
    "multiqc": {
        "content": [
            [
                "multiqc/multiqc_data/BETA-multiqc.parquet",
                "multiqc/multiqc_data/bowtie2_pe_plot.yaml",
                "multiqc/multiqc_data/busco_plot_bacteria_odb10.yaml",
                "multiqc/multiqc_data/fastp-insert-size-plot.yaml",
                "multiqc/multiqc_data/fastp-seq-content-gc-plot_Read_1_After_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-content-gc-plot_Read_1_Before_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-content-gc-plot_Read_2_After_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-content-gc-plot_Read_2_Before_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-content-n-plot_Read_1_After_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-content-n-plot_Read_1_Before_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-content-n-plot_Read_2_After_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-content-n-plot_Read_2_Before_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-quality-plot_Read_1_After_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-quality-plot_Read_1_Before_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-quality-plot_Read_2_After_filtering.yaml",
                "multiqc/multiqc_data/fastp-seq-quality-plot_Read_2_Before_filtering.yaml",
                "multiqc/multiqc_data/fastp_filtered_reads_plot.yaml",
                "multiqc/multiqc_data/fastqc-1-status-check-heatmap.yaml",
                "multiqc/multiqc_data/fastqc-1_per_base_n_content_plot.yaml",
                "multiqc/multiqc_data/fastqc-1_per_base_sequence_quality_plot.yaml",
                "multiqc/multiqc_data/fastqc-1_per_sequence_gc_content_plot_Counts.yaml",
                "multiqc/multiqc_data/fastqc-1_per_sequence_gc_content_plot_Percentages.yaml",
                "multiqc/multiqc_data/fastqc-1_per_sequence_quality_scores_plot.yaml",
                "multiqc/multiqc_data/fastqc-1_sequence_counts_plot.yaml",
                "multiqc/multiqc_data/fastqc-1_sequence_duplication_levels_plot.yaml",
                "multiqc/multiqc_data/fastqc-status-check-heatmap.yaml",
                "multiqc/multiqc_data/fastqc_per_base_n_content_plot.yaml",
                "multiqc/multiqc_data/fastqc_per_base_sequence_quality_plot.yaml",
                "multiqc/multiqc_data/fastqc_per_sequence_gc_content_plot_Counts.yaml",
                "multiqc/multiqc_data/fastqc_per_sequence_gc_content_plot_Percentages.yaml",
                "multiqc/multiqc_data/fastqc_per_sequence_quality_scores_plot.yaml",
                "multiqc/multiqc_data/fastqc_sequence_counts_plot.yaml",
                "multiqc/multiqc_data/fastqc_sequence_duplication_levels_plot.yaml",
                "multiqc/multiqc_data/fastqc_sequence_length_distribution_plot.yaml",
                "multiqc/multiqc_data/multiqc.log",
                "multiqc/multiqc_data/multiqc_bowtie2.yaml",
                "multiqc/multiqc_data/multiqc_bowtie2_bowtie2-1.yaml",
                "multiqc/multiqc_data/multiqc_bowtie2_bowtie2-2.yaml",
                "multiqc/multiqc_data/multiqc_busco.yaml",
                "multiqc/multiqc_data/multiqc_citations.yaml",
                "multiqc/multiqc_data/multiqc_data.json",
                "multiqc/multiqc_data/multiqc_fastp.yaml",
                "multiqc/multiqc_data/multiqc_fastqc.yaml",
                "multiqc/multiqc_data/multiqc_fastqc_fastqc-1.yaml",
                "multiqc/multiqc_data/multiqc_general_stats.yaml",
                "multiqc/multiqc_data/multiqc_prokka.yaml",
                "multiqc/multiqc_data/multiqc_quast.yaml",
                "multiqc/multiqc_data/multiqc_quast_quast-1.yaml",
                "multiqc/multiqc_data/multiqc_software_versions.yaml",
                "multiqc/multiqc_data/multiqc_sources.yaml",
                "multiqc/multiqc_data/porechop.yaml",
                "multiqc/multiqc_data/prokka_plot.yaml",
                "multiqc/multiqc_data/quast_num_contigs.yaml",
                "multiqc/multiqc_data/quast_table.yaml",
                "multiqc/multiqc_plots/pdf/bowtie2_pe_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/bowtie2_pe_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/busco_plot_bacteria_odb10-cnt.pdf",
                "multiqc/multiqc_plots/pdf/busco_plot_bacteria_odb10-pct.pdf",
                "multiqc/multiqc_plots/pdf/fastp-insert-size-plot.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-gc-plot_Read_1_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-gc-plot_Read_1_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-gc-plot_Read_2_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-gc-plot_Read_2_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-n-plot_Read_1_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-n-plot_Read_1_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-n-plot_Read_2_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-n-plot_Read_2_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-quality-plot_Read_1_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-quality-plot_Read_1_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-quality-plot_Read_2_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-quality-plot_Read_2_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp_filtered_reads_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/fastp_filtered_reads_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1-status-check-heatmap.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_per_base_n_content_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_per_base_sequence_quality_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_per_sequence_gc_content_plot_Counts.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_per_sequence_gc_content_plot_Percentages.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_per_sequence_quality_scores_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_sequence_counts_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_sequence_counts_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_sequence_duplication_levels_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-status-check-heatmap.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_base_n_content_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_base_sequence_quality_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_sequence_gc_content_plot_Counts.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_sequence_gc_content_plot_Percentages.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_sequence_quality_scores_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_counts_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_counts_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_duplication_levels_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_length_distribution_plot.pdf",
                "multiqc/multiqc_plots/pdf/prokka_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/prokka_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/quast_num_contigs-cnt.pdf",
                "multiqc/multiqc_plots/pdf/quast_num_contigs-pct.pdf",
                "multiqc/multiqc_plots/pdf/quast_table.pdf",
                "multiqc/multiqc_plots/png/bowtie2_pe_plot-cnt.png",
                "multiqc/multiqc_plots/png/bowtie2_pe_plot-pct.png",
                "multiqc/multiqc_plots/png/busco_plot_bacteria_odb10-cnt.png",
                "multiqc/multiqc_plots/png/busco_plot_bacteria_odb10-pct.png",
                "multiqc/multiqc_plots/png/fastp-insert-size-plot.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-gc-plot_Read_1_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-gc-plot_Read_1_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-gc-plot_Read_2_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-gc-plot_Read_2_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-n-plot_Read_1_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-n-plot_Read_1_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-n-plot_Read_2_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-n-plot_Read_2_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-quality-plot_Read_1_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-quality-plot_Read_1_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-quality-plot_Read_2_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-quality-plot_Read_2_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp_filtered_reads_plot-cnt.png",
                "multiqc/multiqc_plots/png/fastp_filtered_reads_plot-pct.png",
                "multiqc/multiqc_plots/png/fastqc-1-status-check-heatmap.png",
                "multiqc/multiqc_plots/png/fastqc-1_per_base_n_content_plot.png",
                "multiqc/multiqc_plots/png/fastqc-1_per_base_sequence_quality_plot.png",
                "multiqc/multiqc_plots/png/fastqc-1_per_sequence_gc_content_plot_Counts.png",
                "multiqc/multiqc_plots/png/fastqc-1_per_sequence_gc_content_plot_Percentages.png",
                "multiqc/multiqc_plots/png/fastqc-1_per_sequence_quality_scores_plot.png",
                "multiqc/multiqc_plots/png/fastqc-1_sequence_counts_plot-cnt.png",
                "multiqc/multiqc_plots/png/fastqc-1_sequence_counts_plot-pct.png",
                "multiqc/multiqc_plots/png/fastqc-1_sequence_duplication_levels_plot.png",
                "multiqc/multiqc_plots/png/fastqc-status-check-heatmap.png",
                "multiqc/multiqc_plots/png/fastqc_per_base_n_content_plot.png",
                "multiqc/multiqc_plots/png/fastqc_per_base_sequence_quality_plot.png",
                "multiqc/multiqc_plots/png/fastqc_per_sequence_gc_content_plot_Counts.png",
                "multiqc/multiqc_plots/png/fastqc_per_sequence_gc_content_plot_Percentages.png",
                "multiqc/multiqc_plots/png/fastqc_per_sequence_quality_scores_plot.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_counts_plot-cnt.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_counts_plot-pct.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_duplication_levels_plot.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_length_distribution_plot.png",
                "multiqc/multiqc_plots/png/prokka_plot-cnt.png",
                "multiqc/multiqc_plots/png/prokka_plot-pct.png",
                "multiqc/multiqc_plots/png/quast_num_contigs-cnt.png",
                "multiqc/multiqc_plots/png/quast_num_contigs-pct.png",
                "multiqc/multiqc_plots/png/quast_table.png",
                "multiqc/multiqc_plots/svg/bowtie2_pe_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/bowtie2_pe_plot-pct.svg",
                "multiqc/multiqc_plots/svg/busco_plot_bacteria_odb10-cnt.svg",
                "multiqc/multiqc_plots/svg/busco_plot_bacteria_odb10-pct.svg",
                "multiqc/multiqc_plots/svg/fastp-insert-size-plot.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-gc-plot_Read_1_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-gc-plot_Read_1_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-gc-plot_Read_2_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-gc-plot_Read_2_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-n-plot_Read_1_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-n-plot_Read_1_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-n-plot_Read_2_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-n-plot_Read_2_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-quality-plot_Read_1_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-quality-plot_Read_1_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-quality-plot_Read_2_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-quality-plot_Read_2_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp_filtered_reads_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/fastp_filtered_reads_plot-pct.svg",
                "multiqc/multiqc_plots/svg/fastqc-1-status-check-heatmap.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_per_base_n_content_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_per_base_sequence_quality_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_per_sequence_gc_content_plot_Counts.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_per_sequence_gc_content_plot_Percentages.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_per_sequence_quality_scores_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_sequence_counts_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_sequence_counts_plot-pct.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_sequence_duplication_levels_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-status-check-heatmap.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_base_n_content_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_base_sequence_quality_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_sequence_gc_content_plot_Counts.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_sequence_gc_content_plot_Percentages.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_sequence_quality_scores_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_counts_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_counts_plot-pct.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_duplication_levels_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_length_distribution_plot.svg",
                "multiqc/multiqc_plots/svg/prokka_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/prokka_plot-pct.svg",
                "multiqc/multiqc_plots/svg/quast_num_contigs-cnt.svg",
                "multiqc/multiqc_plots/svg/quast_num_contigs-pct.svg",
                "multiqc/multiqc_plots/svg/quast_table.svg",
                "multiqc/multiqc_report.html"
            ],
            [
                "bowtie2_pe_plot.yaml:md5,331d49d9aa0a0ce1a3a8b407b8a49801",
                "busco_plot_bacteria_odb10.yaml:md5,891eb4ee4391f6f36c18c5b82a7d2473",
                "fastp-insert-size-plot.yaml:md5,55f97b7830c1395c10c06bf3f4431186",
                "fastp-seq-content-gc-plot_Read_1_After_filtering.yaml:md5,fb178f8c18fde4ad16d79afe333d481b",
                "fastp-seq-content-gc-plot_Read_1_Before_filtering.yaml:md5,2f3f9726d97d4f22be3f4ad30ee14173",
                "fastp-seq-content-gc-plot_Read_2_After_filtering.yaml:md5,9e0cf9755e26ac69fa76753137b69ed0",
                "fastp-seq-content-gc-plot_Read_2_Before_filtering.yaml:md5,9bce88d288517439d4b7d263e96dec40",
                "fastp-seq-content-n-plot_Read_1_After_filtering.yaml:md5,972e679cfbb39152a39216dd0c17070d",
                "fastp-seq-content-n-plot_Read_1_Before_filtering.yaml:md5,972e679cfbb39152a39216dd0c17070d",
                "fastp-seq-content-n-plot_Read_2_After_filtering.yaml:md5,972e679cfbb39152a39216dd0c17070d",
                "fastp-seq-content-n-plot_Read_2_Before_filtering.yaml:md5,972e679cfbb39152a39216dd0c17070d",
                "fastp-seq-quality-plot_Read_1_After_filtering.yaml:md5,eb064944225683afb953e68ac241c4cd",
                "fastp-seq-quality-plot_Read_1_Before_filtering.yaml:md5,d398e6c77c790e63a266efc0a5569007",
                "fastp-seq-quality-plot_Read_2_After_filtering.yaml:md5,f1f9bd9b2adb1be42654f5f4ec8de619",
                "fastp-seq-quality-plot_Read_2_Before_filtering.yaml:md5,c86d90f038fd107d95caa118a7fed4d1",
                "fastp_filtered_reads_plot.yaml:md5,ef1888ea78cbe95c474e2248da2469ec",
                "fastqc-1-status-check-heatmap.yaml:md5,1300edec360b1ba75c7f454bdd734aee",
                "fastqc-1_per_base_n_content_plot.yaml:md5,0e74e961d060e677c85bc0963c2a22cd",
                "fastqc-1_per_base_sequence_quality_plot.yaml:md5,f03db993986d537a2d75b923259b1b4d",
                "fastqc-1_per_sequence_gc_content_plot_Counts.yaml:md5,03f3fab53bb263f4f01f1a7653b44e59",
                "fastqc-1_per_sequence_gc_content_plot_Percentages.yaml:md5,bc76a48e0ec1d9878a4420fe0854aa7e",
                "fastqc-1_per_sequence_quality_scores_plot.yaml:md5,70e857785001ac547ad0c41efe40df44",
                "fastqc-1_sequence_counts_plot.yaml:md5,919116e2364bc98486240ff29d1a0867",
                "fastqc-1_sequence_duplication_levels_plot.yaml:md5,6b127287a3077610e4ef19965c00e91b",
                "fastqc-status-check-heatmap.yaml:md5,995feb18817c87be186d0e3ec27d826d",
                "fastqc_per_base_n_content_plot.yaml:md5,b0a60ab64b17b73ce2000f5d5facfd5c",
                "fastqc_per_base_sequence_quality_plot.yaml:md5,72619016315ba2c71ba7c96e670e0190",
                "fastqc_per_sequence_gc_content_plot_Counts.yaml:md5,f33c4fc162816aa3657472d51d547c67",
                "fastqc_per_sequence_gc_content_plot_Percentages.yaml:md5,6e797e0817c813313f481fce823fe567",
                "fastqc_per_sequence_quality_scores_plot.yaml:md5,e0ac4f1b3b6e82ae3d8e4191055b67d1",
                "fastqc_sequence_counts_plot.yaml:md5,65750c5392670ff6a7237dc7f097e077",
                "fastqc_sequence_duplication_levels_plot.yaml:md5,8d31278516c68ee70086d2ac0880b394",
                "fastqc_sequence_length_distribution_plot.yaml:md5,24576ce885728df6abaa003cca40b9cb",
                "multiqc_bowtie2.yaml:md5,d6a4f1f2e6952801eda62c414c1685cb",
                "multiqc_bowtie2_bowtie2-1.yaml:md5,d6a4f1f2e6952801eda62c414c1685cb",
                "multiqc_bowtie2_bowtie2-2.yaml:md5,b5a185109453a80131d7da2cf4d3f2ec",
                "multiqc_busco.yaml:md5,76cda2e3678fbdf6a592da3bb7fcdc37",
                "multiqc_citations.yaml:md5,49f79471c1620b06da08b1acb97a1e56",
                "multiqc_fastp.yaml:md5,5d2c6eefa228bc0210be0d017eeb8eda",
                "multiqc_fastqc.yaml:md5,0d11a6d30d0d1a430e58d44b57ac6e87",
                "multiqc_fastqc_fastqc-1.yaml:md5,0b8c8b4edb4f3e70409f30bcdda54397",
                "multiqc_general_stats.yaml:md5,f0c8018902770fbf04fb1c7d813fae8c",
                "multiqc_prokka.yaml:md5,f7ee4045a273845b501ba15e5fd4d38f",
                "multiqc_quast.yaml:md5,e5940a0ce6af05da51fe94bc6d7d3901",
                "multiqc_quast_quast-1.yaml:md5,4fcb2008af9fc15990854d76e6a72ada",
                "multiqc_software_versions.yaml:md5,60f40217532e70761e9b2ea2c601aef3",
                "porechop.yaml:md5,828d96e2822bb4b4108ab706c00c4450",
                "prokka_plot.yaml:md5,229d29dafbf6bde37d8f245dfb535b2e",
                "quast_num_contigs.yaml:md5,241f678b1df1c1dd0dd58246a073d7dc",
                "quast_table.yaml:md5,c1c9b5770b5defb3595b7b46af6a746c"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.7"
        },
        "timestamp": "2025-09-12T09:08:45.630675675"
    }
}